environment_and_booting.md

```diff
@@ -132,11 +132,13 @@ describes how to set one up). Save the following code in a file called
     global loader                   ; the entry symbol for ELF
 
     MAGIC_NUMBER equ 0x1BADB002     ; define the magic number constant
-    CHECKSUM     equ -MAGIC_NUMBER  ; calculate the checksum (magic number + checksum should equal 0)
+    FLAGS        equ 0x0            ; multiboot flags
+    CHECKSUM     equ -MAGIC_NUMBER  ; calculate the checksum (magic number + checksum + flags should equal 0)
```

Comment:
Next commit address this issue. HTML output looks ok now in Firefox.

In Chrome I have a scrollbar, but it is caused by another line, not by the checksum line. Probably because I'm using customized font settings.